### ISSUE TITLE
Indicate who is speaking?

### ISSUE THREAD

**Issue by johnbburk** [arundquist/synchronous-dashboard-with-jitsi#1]
Right now, I don't see how to tell who is speaking. This is challenging in breakout rooms where I'd like to get to know the people in my group. I'm wondering if it's possible to display a mic icon next to the name of the person in the list, or perhaps put a list of people next to the mic button who have their microphones on.

**Comment by arundquist** [comment]
yes I think [this command](https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-iframe#dominantspeakerchanged) will do the trick
